### ISSUE TITLE
Fix for Bootstrap 4.1.3 and Jquery 3.3.1

### ISSUE THREAD

**Issue by federico-66** [saribe/eModal#85]
Hi,
with these fix, eModal works with Bootstrap 4.1.3 and Jquery 3.3.1

line 233, replace
```
'<div class=modal-header><button type=button class="x close" data-dismiss=modal><span aria-hidden=true>&times;</span><span class=sr-only>Close</span></button><h4 class=modal-title></h4></div>' +
```
with
```
'<div class=modal-header><h5 class="modal-title">Modal title</h5><button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button></div>' +
```
line 236, replace
```
return $('<div class="modal fade" tabindex="-1"><style>.modal-xl{width:96%;}.modal-body{max-height: calc(100vh - 145px);overflow-y: auto;}</style>' +
```
with
```
return $('<div class="modal fade" tabindex="-1"><style>@media(min-width:992px){.modal-xl{max-width:none;width:96%;}}.modal-body{max-height:calc(100vh-145px);overflow-y:auto;}</style>' +
```
line 459, replace
```
var message = $(html).load(iframeReady);
```
with
```
var message=$(html).on('load', iframeReady);
```

**Comment by matsn0w** [comment]
Made a pull request for this, thanks!

**Comment by saribe** [comment]
Fix on v1.2.66
